templates/CRM/Custom/Form/Preview.tpl

```diff
@@ -62,11 +62,10 @@
          </td>
         </tr>
   {else}
-        {assign var="name" value=`$element.name`}
-        {*assign var="element_name" value=$group_id|cat:_|cat:$field_id|cat:_|cat:$element.name*}
-        {assign var="element_name" value=$element.element_name}
+        {capture assign="name"}{if !empty($element.name)}{$element.name}{/if}{/capture}
+        {capture assign="element_name"}{if !empty($element.element_name)}{$element.element_name}{/if}{/capture}
         <tr>
```

Comment:
These are interesting. I guess it could also be done the same way as in ActivityLinks.tpl above.

And maybe a future followup I don't see where the first one - $name - is used.